app/models/collection.rb

```diff
@@ -156,7 +156,7 @@
     too_long: ts("must be less than %{max} characters long.", max: ArchiveConfig.SUMMARY_MAX)
 
   validates_format_of :header_image_url, allow_blank: true, with: URI::regexp(%w(http https)), message: ts("is not a valid URL.")
-  validates_format_of :header_image_url, allow_blank: true, with: /\.(png|gif|jpg)$/, message: ts("can only point to a gif, jpg, or png file."), multiline: true
+  validates_format_of :header_image_url, allow_blank: true, with: /\.(png|gif|jpg|jpeg)$/, message: ts("can only point to a gif, jpg, jpeg or png file."), multiline: true
```

Comment:
Huh. That's a new one. I'll be looking into this.

Reply:
If you haven't had a chance to dig in, https://stackoverflow.com/a/577675 has some good information (I had it handy because we made a similar change in https://github.com/otwcode/otwarchive/pull/4601).

TLDR - if someone changes the input to be multiline (like a textarea) they could, in theory enter something like
```
sneaky.jpeg
haha, another line
```
IIRC in most languages, `$` and `\z` end up being functionally equivalent, but not in Ruby

Reply:
Thanks! I haven't had the chance thanks to life. I appreciate the information to work on this!